buildtools/npm/manifest_test.go

```diff
@@ -1,65 +1,111 @@
 package npm_test
 
 import (
-	"errors"
+	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
 	"github.com/fossas/fossa-cli/buildtools/npm"
+	"github.com/fossas/fossa-cli/pkg"
 )
 
+func getDirectDepImports() (chaiDirectDep pkg.Import, typeDetectDirectDep pkg.Import) {
```

Comment:
Again, why not make these package variables? There's no advantage to making this a function. I would prefer using the language construct that communicates intent most clearly rather than the technically safest one.